mutual_debt/tests/test_mutual_debt_simplification.py

```diff
@@ -1,5 +1,7 @@
 import unittest
-from mutual_debt_simplification import simplify_debt_graph, debt_list_to_graph
+
+from mutual_debt.mutual_debt_simplification import simplify_debt_graph, \
```

Comment:
`mutual_debt.mutual_debt_simplification` is a bit redundant, the module could be changed to `simplification`

Reply:
Totally agree